test/AutoRest.TestServer.Tests/lro.cs

```diff
@@ -0,0 +1,1455 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using AutoRest.TestServer.Tests.Infrastructure;
+using Azure;
+using Azure.Core;
+using lro;
+using lro.Models;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests
+{
+    [IgnoreOnTestServer(TestServerVersion.V2, "LRO tests are not supported yet")]
+    public class LroTest: TestServerTestBase
+    {
+        public LroTest(TestServerVersion version) : base(version, "lros") { }
+
+        [Test]
+        [Ignore("Needs to send x-ms-client-request-id: https://github.com/Azure/autorest.csharp/issues/446")]
```

Comment:
I think I asked you about this already but I forgot. Why is this skipped? The full pipeline should be sending the `x-ms-client-request-id`

Reply:
You have to send a specific value for `x-ms-client-request-id`. Is there a way to inject a value per request (that won't be overridden)?

Reply:
It sends a random one per request. Do we need a specific one for every request?

Reply:
For these tests, yes. There is a value on the test server that needs to be sent to it. So, it cannot be random.